### DISCUSSION TITLE
2024-02-14 Weekly Meeting 146 Notes/Agenda

### DISCUSSION THREAD

**Discussion by indiebio** [omigroup/omigroup#442] [Weekly Meetings]
Weekly Meeting 146 is happening on Wednesday, 14 February, 2024 from 9:00 AM PDT (17:00 UTC) - 10:00 AM PDT (18:00 UTC)

You can add it to your calendar [here](https://calendar.google.com/calendar/u/1?cid=Y18wZHB1Z2Y5ZjgzZXE0cWVrbWI2b21xYmptZ0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t).

We'll be meeting in the NEW SERVER, in the omi-weekly-meeting channel of the [OMI Discord](https://discord.gg/rrHfT7H3).
We regularly use the # omi-meeting-chat channel for text chat.

Happy I love something day! Let's all share something we love. Bonus points for funny, weird, or totes random.

### Agenda

- New (improved) logo conversation, vote
- [conversation in Discord](https://discord.com/channels/887789163045335081/887789163045335086/1206354930365890590)
![image](https://github.com/omigroup/omigroup/assets/23474368/10898e62-2387-47ef-9156-27008e6fc3db)

Related: blog about the logo and how we interpret it (very rough draft, based on the conversation on Discordhttps://hackmd.io/@indiebio/omi-logo)

Working group check ins
- glTF
- publicity
-  backlog meeting
Demos and discussion

**Comment by indiebio** [comment]
Publicity item: Blog and mailing list draft and order of sending ( discussion will probably mostly happen in Publicity meeting Tuesday (the day before)

**Comment by mrmetaverse** [comment]
New backlog is squeaky clean! https://github.com/orgs/omigroup/projects/7

**Reply by indiebio** [reply]
Backlog comment: ideally work on one ticket at a time, and ideally one person only on a ticket (or as arranged - talk not surprise. Collaborate, not steamroller. We value improvement, not ego.)
This is a tool for communication, not policing :)

**Reply by mrmetaverse** [reply]
Working agreement basics: 
- anyone can contribute to the backlog. 
  - comments can be converted to task items from discussions and other tickets/issues on the backlog. 
  - if you are looking for ways to help, check the backlog to do column first to ensure you are not duplicating work, and the necessary people who want to help can help. 
- if you are going to pick up a ticket, leave a comment on the ticket and move it to in progress. 
- if you want to help with a ticket that is owned, please drop a comment on that ticket letting the owner know. 
- default to prioritizing progress, and communicate kindly 
- assuming positive intent. 
- the work item should have a clear definition of done before moving from unplanned to to-do.

**Reply by mrmetaverse** [reply]
use a task that was created yesterday in omi media as an example for adding items to the backlog.

**Reply by mrmetaverse** [reply]
to do: please add the remaining tasks to the media backlog and sync with holistic

**Comment by indiebio** [comment]
Media meeting from yesterday: https://github.com/omigroup/media/discussions/70

**Comment by mrmetaverse** [comment]
## New Logo chat 

problem solved: the original logo is hard to see as a discord logo.
solution: this new one is easier to see, while holding the same general elements 

the logo reflects how agreed upon shapes, and the space  between, make for a better outcome.

**Reply by mrmetaverse** [reply]
lets vote! Keep the old, or migrate to the new (and optimize if needed) 
![image](https://github.com/omigroup/omigroup/assets/63426722/c4437903-898a-4b0f-a610-cda5de6e9665)

**Reply by mrmetaverse** [reply]
figma space https://www.figma.com/file/7IkelFTsQPpMwzrWGzqFBA/Omigroup-Designs?type=design&node-id=363%3A115&mode=design&t=0aDSm4rcSw6c2R7w-1

**Reply by mrmetaverse** [reply]
vote passed! switch the logo on all channels
![image](https://github.com/omigroup/omigroup/assets/63426722/1f54cab9-45a3-46e1-b488-c57c5ee1dd30)

**Reply by indiebio** [reply]
make a new/updated style guide
